sigma.py

```diff
@@ -1,7 +1,8 @@
 import json
 import os
 from typing import Dict, Any
-import yaml
+import cProfile
```

Comment:
testing/profiling import left in

Reply:
addressed
https://github.com/CybercentreCanada/assemblyline-service-sigma/pull/1/commits/56ab181b084601f6fca83ce860ae1982a7e4d3c1